documentation/modules/ROOT/pages/setup.adoc

```diff
@@ -105,8 +105,7 @@ minikube profile knative \#<1>
 minikube start -p knative --memory=8192 --cpus=4 \#<2> 
   --kubernetes-version=v1.12.0 \#<3> 
   --vm-driver=hyperkit \#<4>  
-  --disk-size=50g \
-  --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
```

Comment:
why was this deleted? IIRC only `MutatingAdmissionWebhook` might become default with latest k8s.

Reply:
Because only NamespaceExists is not on the list of enabled by default admission controllers, the rest of them are: https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#which-plugins-are-enabled-by-default. And things seem to work just fine even without NamespaceExists mentioned.

Reply:
@kameshsampath I didn't need it for my run through the tutorial, but if we'd like to keep them explicitly listed that's fine